packages/strapi/package.json

```diff
@@ -103,5 +85,48 @@
     "npm": ">= 6.0.0"
   },
   "license": "MIT",
-  "gitHead": "c85658a19b8fef0f3164c19693a45db305dc07a9"
+  "gitHead": "c85658a19b8fef0f3164c19693a45db305dc07a9",
+  "keywords": [
+    "strapi",
+    "cms",
+    "cmf",
+    "content management system",
+    "content management framework",
+    "admin panel",
+    "dashboard",
+    "api",
+    "auth",
+    "framework",
+    "http",
+    "json",
+    "koa",
+    "koajs",
+    "lusca",
+    "mvc",
+    "oauth",
+    "oauth2",
+    "orm",
+    "realtime",
+    "rest",
+    "restful",
+    "security",
+    "jam",
+    "jamstack",
+    "javascript",
+    "headless",
+    "serverless",
+    "MongoDB",
+    "MySQL",
+    "MariaDB",
+    "PostgreSQL",
+    "SQLite",
+    "graphqL",
+    "infrastructure",
+    "backend",
+    "open source",
+    "self hosted",
+    "javascript",
+    "lerna",
+    "lernajs"
+  ],
```

Comment:
Is it possible to also add `react` in the keywords array?

Reply:
Of course, I am on it!